runelite-client/src/main/java/net/runelite/client/plugins/statusorbs/StatusOrbsOverlay.java

```diff
@@ -51,24 +57,32 @@
 
 public class StatusOrbsOverlay extends Overlay
 {
+
+	@Inject
+	private ItemManager itemManager;
+
 	private static final Color HITPOINTS_COLOR = brighter(0x9B0703);
 	private static final Color SPECIAL_COLOR = brighter(0x1E95B0);
 	private static final Color RUN_COLOR = new Color(255, 215, 0);
 	private static final Color OVERLAY_COLOR = new Color(255, 255, 255, 60);
 	private static final double DIAMETER = 26D;
 	private static final int OFFSET = 27;
-
 	private final Client client;
 	private final StatusOrbsPlugin plugin;
 	private final TooltipManager tooltipManager;
-
+	private final HashMap<String, Integer> weapons;
```

Comment:
There's enums in game which map item id -> special attack % and special description. There's no reason whatsoever to use strings for things like this, nor is there to have a map to map everything together.

You should be able to find the enums in [this script](https://github.com/RuneStar/cs2-scripts/blob/master/scripts/%5Bproc,combat_interface_sp_redraw%5D.cs2)